dishdive/components/map.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import { GoogleMap, Marker, LoadScript } from '@react-google-maps/api';
+
+const MapComponent = ({ location }) => {
+  const [mapCenter, setMapCenter] = useState({ lat: 0, lng: 0 });
+
+  useEffect(() => {
+    if (location && location.lat && location.lng) {
+      setMapCenter({ lat: location.lat, lng: location.lng });
+    }
+  }, [location]);
+
+  return (
+    <LoadScript googleMapsApiKey="AIzaSyDVVElh61AtQymBBzyx8S-U42gvIGBrYIU">
```

Comment:
load the key from env
```suggestion
    <LoadScript googleMapsApiKey="{mapApiKey}">
```